config/jobs/etcd/etcd-periodics.yaml

```diff
@@ -1,4 +1,38 @@
 periodics:
+- name: ci-etcd-e2e-arm64
+  interval: 4h
+  cluster: k8s-infra-prow-build
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+    - org: etcd-io
+      repo: etcd
+      base_ref: main
+  annotations:
+    testgrid-dashboards: sig-etcd-periodics, sig-etcd-arm64
+    testgrid-tab-name: ci-etcd-e2e-armd64
+  spec:
+    containers:
+    - image: us-central1-docker.pkg.dev/k8s-staging-test-infra/images/kubekins-e2e:v20240705-131cd74733-master
```

Comment:
Are we asking folks to directly refer to `us-central1-docker.pkg.dev/k8s-staging-test-infra` ?

Reply:
kubekins-e2e-v2 is published here. kubekins-e2e is published at `gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240803-cf1183f2db-master` which is deprecated and should only be used by repos that need the legacy tooling. etcd doesn't need the legacy tools and also needs the arm image.